src/main/scala/org/ergoplatform/nodeView/ErgoInterpreter.scala

```diff
@@ -0,0 +1,61 @@
+package org.ergoplatform.nodeView
+
+import org.ergoplatform.ErgoLikeContext.Height
+import org.ergoplatform._
+import org.ergoplatform.settings.{Constants, Parameters}
+import sigmastate.SBoolean
+import sigmastate.Values.Value
+import sigmastate.interpreter.Interpreter.VerificationResult
+import scala.util.Try
+
+
+class ErgoInterpreter(override val maxCost: Long = Parameters.MaxBlockCost)
```

Comment:
Class comments describing it's purposes might be useful here

Reply:
added